docs/angular/api-react/schematics/application.md

```diff
@@ -48,6 +48,14 @@ ng g app myapp --routing
 
 ## Options
 
+### babel
```

Comment:
Maybe we should call this option `babelJest` since we always use babel for building the application.

Reply:
I have been thinking the same thing 😆